internal/jstest/jstest.go

```diff
@@ -18,18 +18,31 @@ import (
 	"context"
 	"log"
 	"os/exec"
+	"path/filepath"
 	"strings"
 
 	"github.com/FerretDB/dance/internal"
 )
 
-func Run(ctx context.Context, args []string) (*internal.TestResults, error) {
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
```

Comment:
`dir` is not used

Reply:
Yes https://github.com/FerretDB/dance/pull/293/commits/a5ac0416f29843f7a6e037c61533c696b7772d69 added confusion, my bad. I removed `dir` and it works.